data/sql/updates/pending_db_world/rev_1583667245426891200.sql

```diff
@@ -74,10 +73,10 @@ UPDATE `creature_template` SET `mindmg` = 47, `maxdmg` = 63, `DamageModifier` =
 
 -- Shadowfang Keep (22-30)
 -- -------------------------------------------------------------------------------------------------------------------------------------------------
--- Fel Steed (Not listed in Beastiary) Src: https://wotlk.evowow.com/?npc=3864 / https://db.rising-gods.de/?npc=3864
-UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1 WHERE `entry` = 3864;
--- Shadow Charger (Not listed in Beastiary) Src: https://wotlk.evowow.com/?npc=3865 / https://db.rising-gods.de/?npc=3865
-UPDATE `creature_template` SET `mindmg` = 203, `maxdmg` = 288, `DamageModifier` = 1 WHERE `entry` = 3865;
+-- Fel Steed (Not listed in Beastiary) Src: https://classicdb.ch/?npc=3864
+UPDATE `creature_template` SET `mindmg` = 154, `maxdmg` = 183 WHERE `entry` = 3864;
+-- Shadow Charger (Not listed in Beastiary) Src: https://classicdb.ch/?npc=3865
+UPDATE `creature_template` SET `mindmg` = 145, `maxdmg` = 172 WHERE `entry` = 3865;
```

Comment:
as i told you, these are not bosses

Reply:
in fact the shadow charger mob has no spawn ingame wtf (fel steed is a mini boss so it's fine)